.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: false
+
+language: python
+
+python:
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - "pypy"
+  - "pypy3"
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+
+addons:
+  apt:
+    sources:
+      - build-essential
+
+install:
+ - pip install --upgrade pip
+ - pip install --install-option="--no-cython-compile" Cython 
+ - pip install .
+
+script:
+ - cd `mktemp -d`
+ - nosetests -v nxlemon
```

Comment:
Yes, that's what happens in networkx. Isn't it? you run `$ nosetests -v` on the top and it searches for all the tests present in the entire package. That's why we name it like `test_xyz.py` to make it easier for `nose` to search.